elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -150,6 +155,21 @@ class PfeContentSet extends PFElement {
     if (window.ResizeObserver) this._resizeObserver.disconnect();
   }
 
+  fetchComponent() {
```

Comment:
Due to the duplicate ID issue, I went back to the old approach of creating a new component and deleting the old one if the rendering mode changes.